sdk/identity/identity/test/manual/dist/index.html

```diff
@@ -1,6 +0,0 @@
-<html>
```

Comment:
was this removal intentional? @sadasant

Reply:
Yes, this file gets automatically generated.